src/ui_ng/lib/src/tag/tag.component.ts

```diff
@@ -217,7 +217,13 @@ export class TagComponent implements OnInit, AfterViewInit {
     st.page.size = this.pageSize;
     st.page.from = 0;
     st.page.to = this.pageSize - 1;
-    st.filters = [{property: "name", value: this.lastFilteredTagName}];
+    let selectedLab = this.imageFilterLabels.find(label => label.iconsShow === true);
```

Comment:
The variable name of selectedLab may make some confusion. Consider using full name `selectedLabel` instead?

Reply:
Thanks for your advice . I will take care for the next time ..